projects/fsastorefrontlib/src/cms-components/my-account/claim/guards/claim-policies-guard.ts

```diff
@@ -25,6 +25,7 @@ export class ClaimPoliciesGuard implements CanActivate, OnDestroy {
     this.subscription = this.claimService
       .getClaimPolicies()
       .pipe(
+        take(1),
```

Comment:
Not associated directly with the original bug.
Without this, recursion for fetching claim policies will happen once the anonymous user tries to create claim.
![image](https://user-images.githubusercontent.com/71004931/103343006-41d01e80-4a8b-11eb-8b27-36a31cddde19.png)